content/partials/types/_channel_state.textile

```diff
@@ -120,3 +120,18 @@ blang[go].
       StateChanFailed = 32768
     )
   ```
+
+blang[flutter].
+  @ably.ChannelState@ is an enum representing all the "@Realtime Channel@ states":/realtime/channels#channel-states.
+
+  ```[flutter]
+  enum ChannelState {
```

Comment:
Indentation is perhaps two columns too short, to match the level of the preceding Go example, as well as your Flutter code [here](https://github.com/ably/docs/pull/989/files#diff-a35132c3750f7cf9ed3ee4a13e441713b4a8c3ccd3453ae5de5245b1330ba078R122).

Reply:
Addressed 19a969ae337a59f8d8c3c8d8d4bcde6858dbd91c